modules/programs/ssh.nix

```diff
@@ -370,6 +414,25 @@ in
   };
 
   config = mkIf cfg.enable {
+    assertions = [
+      {
+        assertion =
+          let
+            # `builtins.all`/`lib.lists.all` does not return `true` if there are no elements.
+            all' = pred: items: if (builtins.length items == 0) then true else (any pred items);
```

Comment:
Shouldn't this have `all pred items` instead of `any pred items`?

Also, instead of `builtins.length items == 0` it is faster and more concise to do `items == []`.

Reply:
I think it should have been `any`, I just forgot to rename `all'` from an earlier version.